packages/build/tests/plugins/snapshots/tests.js.md

```diff
@@ -8395,7 +8395,7 @@ Generated by [AVA](https://ava.li).
     ────────────────────────────────────────────────────────────────␊
     ␊
       Error message␊
-      Error: "netlifyConfig.Symbol(test)" is read-only.␊
+      Error: "netlifyConfig.*" is read-only.␊
```

Comment:
This is due to symbols being lost during JSON serialization. This is not a problem since symbols are not currently used in the configuration and are unlikely to be used in the future.